### PR TITLE
Update dependencies and apply coding conventions

### PR DESCRIPTION
**Pull request by kaelig**

### CODE REVIEW

**Comment by kaelig** [comment]
@AlbertoElias not sure why this isn't building, can you have a look?

File: package.json [hunk 1]
```diff
@@ -1,17 +1,17 @@
...
-        "phantomjs": "^1.9.7-3",
-        "browserify": "^3.41.0",
-        "karma": "^0.12.6",
-        "karma-jasmine": "^0.1.5",
-        "karma-phantomjs-launcher": "^0.1.4",
-        "karma-browserify": "^0.2.0",
-        "debowerify": "^0.7.0"
-    }
+  "name": "o-squishy-list",
+  "version": "0.0.0",
```
Comment (AlbertoElias) [inline package.json]: Remove the version, it makes the tests fail

Comment (triblondon) [inline package.json]: I think the spec might require a 0.0.0 version - worth checking.

Comment (AlbertoElias) [inline package.json]: http://origami.ft.com/docs/component-spec/modules/

It says that it shouldn't include it

-- 

------------------------------
This email was sent by a company owned by Pearson plc, registered office at 
80 Strand, London WC2R 0RL.  Registered in England and Wales with company 
number 53723.

**Comment by kaelig** [comment]
I've reverted to using PhantomJS 1.9 and it now builds flawlessly :)

**Comment by AlbertoElias** [comment]
Why is the data attribute `data-o-squishy-list--ignore="true"` removed in demos? Rest is fine

**Comment by kaelig** [comment]
Good catch, apparently the demos were generated with a source that was different from the one that's on the repo. It's not needed.

But the weird thing is that it's not being tested…

**Pull request merged**